runtime/base/base.Dockerfile

```diff
@@ -226,7 +226,7 @@ RUN set -xe; \
 # #   - libssh2
 # # Needed by:
 # #   - php
-ENV VERSION_CURL=7.76.0
+ENV VERSION_CURL=7.80.0
```

Comment:
Fixes:




Flaw | From version | To and including | CVE | CWE
-- | -- | -- | -- | --
STARTTLS protocol injection via MITM | 7.20.0 | 7.78.0 | CVE-2021-22947 | CWE-349: Acceptance of Extraneous Untrusted Data With Trusted Data
Protocol downgrade required TLS bypassed | 7.20.0 | 7.78.0 | CVE-2021-22946 | CWE-325: Missing Cryptographic Step
UAF and double-free in MQTT sending | 7.73.0 | 7.78.0 | CVE-2021-22945 | CWE-415: Double Free
CURLOPT_SSLCERT mixup with Secure Transport | 7.33.0 | 7.77.0 | CVE-2021-22926 | CWE-295: Improper Certificate Validation
TELNET stack contents disclosure again | 7.7 | 7.77.0 | CVE-2021-22925 | CWE-457: Use of Uninitialized Variable
Bad connection reuse due to flawed path name checks | 7.10.4 | 7.77.0 | CVE-2021-22924 | CWE-295: Improper Certificate Validation
Metalink download sends credentials | 7.27.0 | 7.77.0 | CVE-2021-22923 | CWE-522: Insufficiently Protected Credentials
Wrong content via metalink not discarded | 7.27.0 | 7.77.0 | CVE-2021-22922 | CWE-20: Improper Input Validation
TLS session caching disaster | 7.75.0 | 7.76.1 | CVE-2021-22901 | CWE-416: Use After Free
TELNET stack contents disclosure | 7.7 | 7.76.1 | CVE-2021-22898 | CWE-457: Use of Uninitialized Variable
schannel cipher selection surprise | 7.61.0 | 7.76.1 | CVE-2021-22897 | CWE-488: Exposure of Data Element to Wrong Session